lib/script_helper.rb

```diff
@@ -0,0 +1,9 @@
+require 'yaml'
+
+module ScriptHelper
+  def self.read_config(binding)
+    YAML.load_file(__dir__+'/../scripts/config.yml').each do |key, value|
+      binding.local_variable_set(key.to_sym, value)
+    end
+  end
+end
```

Comment:
@mccalluc if i understand this correctly, it loads all values from `scripts/config.yml` as local variables wherever `ScriptHelper.read_config(binding)` is called?

If that's the case i'm wondering what is the value of having config values as locally defined variables in the current binding, as opposed to just doing something like `config["config_key"]`)? Defining local variables outside of the local scope seems odd. Also, mightn't you accidentally overwrite a variable?